test/unit/spec/common/cellComponents/fieldTableCellWidget-Spec.js

```diff
@@ -96,7 +96,11 @@ define([
             });
 
             const appSpec = Spec.make({
-                appSpec: model.getItem('app.spec'),
+                appSpec: model.getItem([
```

Comment:
There's a separate spec for each app that gets stored, so the tests need to fetch just one based on the test data.